docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness.md

```diff
@@ -1,8 +1,153 @@
+
+Managing missing data involves:
+
+1. Identifying where data is missing
+2. Understanding the causes of the missing data
+3. Deciding on appropriate remedial actions
+
+GX is an open-source data quality tool that provides powerful features for managing and validating data integrity, including handling missing data. Supported by an active community, GX allows you to create customizable checks, known as Expectations, that help maintain high data quality standards across diverse workflows.
+
+In this guide, we will walk you through utilizing GX to proficiently handle missing data, ensuring your datasets remain clean, consistent, and ready for analysis or machine learning applications. You will learn step-by-step methods for writing Expectations, setting up automated checks, and making informed decisions about managing various types of missing data.
+
+## Prerequisite knowledge
+
+Before diving into this guide, ensure you have a basic understanding of GX components and workflows. If you're new to GX, we recommend starting with our [GX Overview](/core/introduction/gx_overview.md) to familiarize yourself with key concepts and setup procedures.
+
+## Data preview
+
+Below is a sample of the dataset referenced throughout this guide:
+
+| type     | sender_account_number  | recipient_fullname | transfer_amount | transfer_date       | errors |
+|----------|------------------------|--------------------|-----------------|---------------------|--------|
+| domestic | 244084670977           | Jaxson Duke        | 9143.40         | 2024-05-01 01:12    | NULL   |
+| NULL     | 123456789012           | Jane Smith         | 5000.00         | NULL                | NULL   |
+
+This dataset includes potential for missing data in `type` and `transfer_date` columns.
+
+You can [access this dataset](https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/learn_data_quality_use_cases/missingness.csv) from the `great_expectations` GitHub repo in order to reproduce the code recipes provided in this article.
+
+
+## Key missingness Expectations
+
+GX provides a suite of missingness-focused Expectations to manage missing data in your datasets. These Expectations can be added to an Expectation Suite via the GX Cloud UI or using the GX Core Python library.
+
+
+:::[TODO]
+> Add animated GIF demonstrating how to create Expectations using the GX Cloud UI.
+:::
+
+### Expect Column Values To Be Null
+
+Ensures that values within a column are `NULL`.
+
+**Use Case**: Handle columns that should typically be populated but might be left null due to specific conditions.
+
+```python title="" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness_resources/missingness_expectations.py ExpectColumnValuesToBeNull"
+gxe.ExpectColumnValuesToBeNull('errors')
```

Comment:
For the code snippet inclusion in the markdown file, you don't need to include the code in the .md - just the snippet name from the .py file. As an example, this snippet can be updated to:

(code block isn't included, Github rendering made it blank in the comment - not useful!) Just update the text in the snippet code block to reflect below):
python title="Python" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness_resources/missingness_expectations.py ExpectColumnValuesToBeNull"